cli/magic-config.ts

```diff
@@ -546,6 +550,15 @@ async function processCreateOptions(options: any): Promise<void> {
       message: "Do you want to enable RAG",
       initial: options.enableRag || false,
     },
+    {
+      type: "confirm",
+      name: "enableEmbeddingModelsViaSagemaker",
+      message: "Do you want to enable embedding models via SageMaker?",
```

Comment:
For now I suggest to change it to "Do you want to enable embedding and cross-encoder models via SageMaker?" to clarify the consequences of the choice to the users.

When the  PR to enable cohere cross encoders via Cohere API will be merged, we will  add a separate question about enabling cross-encoders for hybrid search.